charts/cray-etcd-backup/files/list_backups

```diff
@@ -15,12 +38,15 @@ def main():
     s3 = boto3.client('s3', endpoint_url=s3_endpoint)
 
     response = s3.list_objects_v2(Bucket=backup_bucket)
-    if 'Contents' in response:
-        for item in response['Contents']:
-            if args.project in item['Key']:
-                print(item['Key'])
-    else:
-        print("No backups available yet.")
+    found_one=False
+    sorted_contents = sorted(response['Contents'], key=lambda d: d['LastModified'])
+    for item in sorted_contents:
+        if args.project in item['Key']:
+            print(item['Key'])
+            found_one=True
```

Comment:
could break out of the for loop after one is found.

Reply:
thanks for the comment -- this intentional to stay in the loop in order to fully list the backups (not just the first one).  the flag is used below if none were found..